apps/src/code-studio/pd/professional_learning_landing/landingPage.jsx

```diff
@@ -2,117 +2,83 @@
-            padding: '30px',
-            fontSize: '40px',
-            color: 'white'
-          }}
-        >
-          My Professional Learning
-        </div>
-      </div>
-    );
+  headerText: {
+    backgroundColor: 'rgba(0, 0, 0, .5)',
+    alignSelf: 'flex-end',
+    width: '100%',
+    textAlign: 'center',
+    padding: '30px',
+    fontSize: '40px',
+    color: 'white'
   },
+};
 
-  renderWorkshopSurveyInterior() {
-    if (this.props.lastWorkshopSurveyCourse === 'CS Fundamentals') {
-      return (
-        <div>
-          <h3>
-            Submit feedback and order free course kit
-          </h3>
-          <p>
-            Thank you for taking a CS Fundamentals workshop! Please complete this survey about your experience and you
-            will be able to order supplies for your classroom.
-          </p>
-          <Button bsStyle="primary" onClick={() => {window.open(this.props.lastWorkshopSurveyUrl, '_blank');}}>
-            Start survey
-          </Button>
-        </div>
-      );
-    } else {
-      return (
-        <div>
-          <h3>
-            Submit your feedback!
-          </h3>
-          <p>
-            Thank you for completing a {this.props.lastWorkshopSurveyCourse} workshop!
-            Please complete this survey about your experience to help us improve future
-            workshops.
-          </p>
-          <Button bsStyle="primary" onClick={() => {window.open(this.props.lastWorkshopSurveyUrl, '_blank');}}>
-            Start survey
-          </Button>
-        </div>
-      );
-    }
-  },
+export default class LandingPage extends Component {
+  static propTypes = {
+    lastWorkshopSurveyUrl: PropTypes.string,
+    lastWorkshopSurveyCourse: PropTypes.string,
+    professionalLearningCourseData: PropTypes.array
+  };
 
   render() {
-    const lastWorkshopSurveyBanner = this.props.lastWorkshopSurveyUrl && (
-      <TwoPartBanner
-        textElement={this.renderWorkshopSurveyInterior()}
-        imageUrl="url('https://code.org/images/email/BJC4NYC.jpg')"
-        imagePosition="imageLeft"
-      />
-    );
-
-    const upcomingWorkshops = (
-      <EnrolledWorkshops
-        key="upcomingWorkshops"
-      />
-    );
-
-    const plcData = !_.isEmpty(this.props.professionalLearningCourseData) && (
-      <ProfessionalLearningCourseProgress
-        professionalLearningCourseData={this.props.professionalLearningCourseData}
-        key="plcData"
-      />
-    );
+    const CSF = this.props.lastWorkshopSurveyCourse === 'CS Fundamentals';
+    const subheading = CSF ?
+      i18n.plLandingSubheadingCSF() :
+      i18n.plLandingSubheading();
+    const description = CSF ?
+      i18n.plLandingDescriptionCSF() :
+      i18n.plLandingDescription();
 
     return (
       <div>
-        {this.renderHeaderImage()}
+        <div style={styles.headerImage}>
+          <div style={styles.headerText}>
+            {i18n.plLandingHeading()}
+          </div>
+        </div>
```

Comment:
Do we prefer inline content over helper functions? I can see advantages to each.

Preferable to both (in my opinion) is having small stateless components that serve the same purpose.  You can even define them later in the file.

```jsx
// here
return (
  <div>
    <HeaderImage/>
    <br />
    //...

// Later in the file

const HeaderImage = () => (
  <div style={styles.headerImage}>
    <div style={styles.headerText}>
      {i18.plLandingHeading()}
    </div>
  </div>
);
```

Reply:
Brent told me we did once a long time ago, but it's very possible that he was referencing a particular implementation in one of my PRs and I over-generalized.  I'll use the small stateless components approach.